.lighthouserc.json

```diff
@@ -63,143 +59,148 @@
             "notification-on-start": ["error", {"minScore": 1}],
             "deprecations": ["error", {"minScore": 1}],
             "password-inputs-can-be-pasted-into": ["error", {"minScore": 1}],
-            "image-aspect-ratio": ["error", {"minScore": 1}]
+            "image-aspect-ratio": ["error", {"minScore": 1}],
+            "offscreen-images": ["error", {"minScore": 0.5}],
+            "is-on-https": ["off", {}],
+            "time-to-first-byte": ["off", {}]
+          }
+        },
+        {
+          "matchingUrlPattern": "http://[^/]+/signup?return_url=%2F$",
+          "assertions": {
+            "uses-webp-images": ["error", {"minScore": 1}]
           }
         },
         {
           "matchingUrlPattern": "http://[^/]+/$",
           "assertions": {
-            "time-to-first-byte": ["off", {"minScore": 1}],
-            "render-blocking-resources": ["error", {"minScore": 0}],
-            "unminified-css": ["error", {"minScore": 0}],
-            "unminified-javascript": ["error", {"minScore": 0}],
-            "unused-css-rules": ["error", {"minScore": 0}],
-            "uses-text-compression": ["error", {"minScore": 0}],
-            "uses-http2": ["error", {"minScore": 0}]
+            "uses-webp-images": ["error", {"minScore": 1}]
+          }
+        },
+        {
+          "matchingUrlPattern": "http://[^/]+/admin$",
+          "assertions": {
+            "uses-webp-images": ["error", {"minScore": 1}]
```

Comment:
Since this has been added to the general section, do we still need it here and other places?

Reply:
It is not in the general section, it is in each page separately because there is some issue in donate page where it sometimes reports lower score. I will investigate this further later, for now lets try to get this PR in as quick as possible since it fixes a lot of security errors.

Reply:
Okay